soap/src/java/com/zimbra/soap/admin/message/GetDeviceStatusRequest.java

```diff
@@ -54,6 +54,13 @@ public class GetDeviceStatusRequest {
     @XmlAttribute(name=SyncConstants.A_LIMIT /* limit */, required=false)
     private int limit = 0;
 
+    /**
+     * @zm-api-field-tag notificationEmail
+     * @zm-api-field-description Indicates if the request is coming from notification email that needs to be sent for all matching conditions across mailbox servers
+     */
+    @XmlAttribute(name=SyncConstants.A_NOTIFICATION_EMAIL /* notificationEmail */, required=false)
```

Comment:
Attribute name `A_NOTIFICATION_EMAIL` is specific to email notification sent for quarantine devices and this API GetDeviceStatusRequest is only specific to device list.
Considering the need of the API to pull all records or with limit, may be can rename to GET_ALL with true Or USE_LIMIT  false to pull all records.

Reply:
done